test/Microsoft.AspNetCore.SignalR.Client.Tests/RpcConnectionFacts.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO.Pipelines;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Sockets.Client;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Microsoft.AspNetCore.SignalR.Client.Tests
```

Comment:
I think we may want to distinguish between FunctionalTests and UnitTests (we do it for instance in Hosting). Because these tests would fall more to the functional tests category.